### PR TITLE
Add better Tuple benchmarks

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

**Comment by nicolasstucki** [comment]
@liufengyun note that I modified I removed the old tuple benchmarks. This set will give us more meaningful results.

**Comment by liufengyun** [comment]
@nicolasstucki Can we just keep the old tests and add new ones? Removing lines involves migrating data, which causes additional maintenance efforts.

**Comment by nicolasstucki** [comment]
Can't we just ignore old data and not show it in the graphs?

**Comment by liufengyun** [comment]
@nicolasstucki We could have stale data in the database, just that it is not ideal. We can merge this anyway, and deal with the obsolete data later.

File: tests/tuple22-tails.scala [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Foo {
+  val tup: (Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int, Int) = ???
+  tup.tail.tail.tail.tail.tail
```
Comment (liufengyun) [inline tests/tuple22-tails.scala]: Do we want to call `head` to mirror `apply`?

Comment (nicolasstucki) [reply tests/tuple22-tails.scala]: Head will become an instance of `apply(0)` therefore the test would be redundant.

**Comment by nicolasstucki** [comment]
It would be good to merge and run it for a few commits in the past to be able to use it in https://github.com/lampepfl/dotty/pull/6539 and compare the results.

File: tests/tuple22-tails.scala (hunk 1 above)
Comment (liufengyun) [reply tests/tuple22-tails.scala]: BTW, do we want to go beyond 22, to test the feature more stressfully :)

Comment (nicolasstucki) [reply tests/tuple22-tails.scala]: Will do. Also should put some small tuples.

Comment (nicolasstucki) [reply tests/tuple22-tails.scala]: Added

**Comment by liufengyun** [comment]
Deployed and scheduled regressional tests for recent 13 merges.

**Pull request merged**